src/policy_artifacthub.rs

```diff
@@ -375,6 +375,10 @@ fn compose_install(oci_url: String) -> String {
 ```console
 kwctl pull {}
 ```
+Then, generate the policy manifest and tune it to your liking. For example:
+```console
+kwctl scaffold manifest -t ClusterAdmissionPolicy registry://{}
```

Comment:
Note that here I needed to add `registry://` while for `kwctl pull` one doesn't need to.
Opened https://github.com/kubewarden/kwctl/issues/537. Maybe worth fixing it at the same time as this for simplicity.